routes/notify.js

```diff
@@ -3,6 +3,7 @@ const router = express.Router();
 
 const authenticate = require("../middlewares/authenticate");
 const { notifyController } = require("../controllers/notify");
+const { notifyValidator } = require("../middlewares/validators/notify");
 
-router.post("/", authenticate, notifyController);
+router.post("/", notifyValidator, authenticate, notifyController);
```

Comment:
will it be a good idea to call authenticate before notifyValidator so that only when a user is authenticated we call notifyValidator

Reply:
updated